mcp/include/command_struct.h

```diff
@@ -68,6 +71,10 @@
 
 /* Need to undef I here in for source files that utilize complex.h */
 #undef I
+/**
+ * @brief PID control gains structure
+ * 
+ */
```

Comment:
If the structure is "eligible" for showing up in the docs (due to the special comment block), for the sake of completeness, I'd describe its members too:
```
struct GainStruct {
    float P;  /**< a brief description of P */
    ...
```
Unless it's just my ignore showing and the meaning of `P` and `I` is obvious.

Reply:
Looked at other files and I guess nevermind. There will be a lot of structures with undocumented members. So for now I only ask that *if* a member already has a description, please make sure it is properly formatted so Doxygen could recognize it.

Reply:
It's proportional and Integral terms in a PID loop, but I'll leave one here. I don't think it will show up, but im not 100%